evcouplings/compare/protocol.py

```diff
@@ -111,29 +139,230 @@ def print_pdb_structure_info(sifts_result, format_string=SIFTS_TABLE_FORMAT_STR,
         **text_kwargs
     )
 
+X_STRUCFREE = [
```

Comment:
this whole section does not really belong into compare - please refactor into a more fitting place (e.g. complex submodule)